framework/db/conditions/BetweenConditionBuilder.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Standard Yii header missing.

Reply:
It will be added during release process automatically